src/utils/sendEmail.js

```diff
@@ -0,0 +1,112 @@
+/*
+ * Takes an email and sends a purchase receipt to that email, including
+ * each item's name, quantity, and price (individual price * quantity), date
+ * of purchase, and order/purchase number
+ * 
+ * Parameters:
+ *    String email: email to send receipt to
+ *    Array item: array of arrays: [name, qty, price]
+ *                (best if price has $ sign) (price is total price of that item)
+ *    String date: date
+ *    String orderNum: order number
+ */
+const fs = require('fs');
+const jsdom = require('jsdom');
+const { JSDOM } = jsdom;
+
+async function sendEmail(email, items, date, orderNum) {
```

Comment:
The documentation above is great. For more robustness, let's use typescript to have the function parameters be typed so we can catch errors on compile time when we call the function.

Typescript is a superset of Javascript, meaning that all Javascript is valid Typescript code. The added benefit with Typescript is that you can now type your elements, so if there are any typing inconsistencies, Typescript will catch it on compile time. To use Typescript, change `sendEmail.js` to `sendEmail.ts`. Here's how to run a typescript file: https://www.geeksforgeeks.org/how-to-execute-typescript-file-using-command-line/#

For your sendEmail parameters (with the change in `items` I mentioned above), the function definition would be `async function sendEmail(email: string, items: [{name: string, qty: number, price: string}], date: Date, orderNum: number)`